Modules/WebSocket/Sources/WebSocket/Frame.swift

```diff
@@ -110,111 +88,120 @@ struct Frame {
         if masked {
             offset += 4
 
-            var unmaskedPayloadData = Data(data[offset ..< data.count])
+            // TODO: remove copy
+            var unmaskedPayloadData = Array(data.subdata(in: offset..<data.count))
```

Comment:
unfortunately suffix and prefix return subsequences which are not of type `Buffer`